modules/tinymce/src/models/dom/main/ts/table/api/TableResizeHandler.ts

```diff
@@ -138,32 +178,32 @@ export const TableResizeHandler = (editor: Editor): TableResizeHandler => {
         sz.on();
       }
 
-      sz.events.startDrag.bind((_event) => {
-        selectionRng.set(editor.selection.getRng());
-      });
+      bindEvents(sz);
+      tableResize.set(sz);
+    }
+  };
 
-      sz.events.beforeResize.bind((event) => {
-        const rawTable = event.table.dom;
-        Events.fireObjectResizeStart(editor, rawTable, Utils.getPixelWidth(rawTable), Utils.getPixelHeight(rawTable), barResizerPrefix + event.type);
-      });
+  editor.on('init', () => createResizeHandler());
 
-      sz.events.afterResize.bind((event) => {
-        const table = event.table;
-        const rawTable = table.dom;
-        Utils.removeDataStyle(table);
+  if (editor.inline) {
+    editor.on('NodeChange', () => {
+      if (isDragging.get()) {
+        return;
+      }
 
-        selectionRng.on((rng) => {
-          editor.selection.setRng(rng);
-          editor.focus();
-        });
+      const targetElm = editor.selection.getNode();
+      const controlElm = SelectorFind.closest<HTMLElement>(SugarElement.fromDom(targetElm), 'table')
+        .map((e) => e.dom)
+        .getOrUndefined();
 
-        Events.fireObjectResized(editor, rawTable, Utils.getPixelWidth(rawTable), Utils.getPixelHeight(rawTable), barResizerPrefix + event.type);
-        editor.undoManager.add();
-      });
+      const isChildOrEqual = (node: Node, parent: Node): boolean =>
```

Comment:
Heh yeah maybe I should have named that function `contains` since it acts like the browser contains function.